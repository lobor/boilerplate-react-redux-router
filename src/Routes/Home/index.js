import React from 'react'

import {
  H1,
  Box,
  Row,
  Column,
  Button,
  Input,
  Select,
	Textarea,
	Nav,
	Alert
} from 'components'

export default class Home extends React.Component{
	render() {
		return (
			<Column>
		    <H1>Hello</H1>
		    <Row>
		      <Column>
						<label htmlFor="email">Input Email</label>
		      	<Input type="email" id="email" name="toto" placeholder="dupont@ymail.com"/>
		      </Column>
		      <Column>
						<label htmlFor="textarea">Textarea</label>
		        <Textarea id="textarea" />
					</Column>
		      <Column>
						<label htmlFor="select">Select</label>
		        <Select id="select">
		          <option>Toto</option>
		        </Select>
					</Column>
		    </Row>
		    <Row left>
		      <Button>Cancel</Button>
		      <Button primary>Valider</Button>
		    </Row>
		    <Row>
		      <Box>Col 1</Box>
		      <Box>Col 2</Box>
		      <Box>Col 3</Box>
		    </Row>
		    <Alert>Alert</Alert>
		    <Alert warning>Alert warning</Alert>
		    <Alert error>Alert error</Alert>
		  </Column>
		)
	}
}
