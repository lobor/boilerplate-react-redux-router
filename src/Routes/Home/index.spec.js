import React from 'react'
import renderer from 'react-test-renderer'

import Home from './index'

describe('Test route Home', () => {
	const tree = renderer.create(<Home />).toJSON()

	test('render Home', () => {
	  expect(tree).toMatchSnapshot()
	})
})
