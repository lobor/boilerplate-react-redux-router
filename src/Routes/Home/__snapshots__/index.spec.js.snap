// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test route Home render Home 1`] = `
<div
  className="sc-bxivhb lluzpC"
>
  <h1
    className="sc-bdVaJa kwHLXg"
  />
  <div
    className="sc-htpNat bBHonn"
  >
    <div
      className="sc-bxivhb lluzpC"
    >
      <label
        htmlFor="email"
      >
        Input Email
      </label>
      <input
        className="sc-EHOje jbbhzM"
        id="email"
        name="toto"
        placeholder="dupont@ymail.com"
        type="email"
      />
    </div>
    <div
      className="sc-bxivhb lluzpC"
    >
      <label
        htmlFor="textarea"
      >
        Textarea
      </label>
      <textarea
        className="sc-gzVnrw fpgqiK"
        id="textarea"
      />
    </div>
    <div
      className="sc-bxivhb lluzpC"
    >
      <label
        htmlFor="select"
      >
        Select
      </label>
      <select
        className="sc-bZQynM bLpLBk"
        id="select"
      >
        <option>
          Toto
        </option>
      </select>
    </div>
  </div>
  <div
    className="sc-htpNat bdtpqn"
  >
    <button
      className="sc-ifAKCX kUckIj"
    >
      Cancel
    </button>
    <button
      className="sc-ifAKCX cYWbib"
    >
      Valider
    </button>
  </div>
  <div
    className="sc-htpNat bBHonn"
  >
    <div
      className="sc-bwzfXH UDjhZ"
    >
      Col 1
    </div>
    <div
      className="sc-bwzfXH UDjhZ"
    >
      Col 2
    </div>
    <div
      className="sc-bwzfXH UDjhZ"
    >
      Col 3
    </div>
  </div>
  <div
    className="sc-iwsKbI loswFL"
  >
    Alert
  </div>
  <div
    className="sc-iwsKbI gcMXUb"
  >
    Alert warning
  </div>
  <div
    className="sc-iwsKbI fqcyIz"
  >
    Alert error
  </div>
</div>
`;
